src/dlangui/dialogs/filedlg.d

```diff
@@ -597,9 +598,9 @@ class FileDialog : Dialog, CustomGridCellAdapter {
         _fileList.layoutWidth(FILL_PARENT).layoutHeight(FILL_PARENT);
         _fileList.resize(4, 3);
         _fileList.setColTitle(0, " "d);
-        _fileList.setColTitle(1, "COL_NAME"d);
-        _fileList.setColTitle(2, "COL_SIZE"d);
-        _fileList.setColTitle(3, "COL_MODIFIED"d);
+        _fileList.setColTitle(1, dtext("COL_NAME"c));
```

Comment:
OK, I found another solution. It's not ideal but for `FileDialog` should be good enough. You can just create `UIString` here from ID and get translated value: `UIString.fromId("COL_NAME"c).value`
Translation will not change when you change language when `FileDialog` is open but that will be better than no translation.